src/mmrelay/message_queue.py

```
@@ -264,6 +324,43 @@ def get_queue_size(self) -> int:
+                new_queue.put_nowait(message)
+                # Move all existing messages to the new queue
+                while not self._queue.empty():
+                    try:
+                        existing_msg = self._queue.get_nowait()
+                        new_queue.put_nowait(existing_msg)
+                        self._queue.task_done()
+                    except (Empty, Full):
+                        break
+                # Replace the queue
+                self._queue = new_queue
+                return True
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)